hack/build-go.sh

```diff
@@ -22,6 +22,7 @@ if [[ -z "$@" ]]; then
 fi
 
 os::build::build_binaries "$@"
+OS_GOFLAGS="${OS_GOFLAGS:-} ${OS_IMAGE_COMPILE_GOFLAGS}" os::build::build_static_binaries "${OS_IMAGE_COMPILE_TARGETS[@]-}" "${OS_SCRATCH_IMAGE_COMPILE_TARGETS[@]-}"
```

Comment:
Without the build of those artifacts, the RPM build fails because those are listed as being present and they would be missing without the cross compile as they are only currently produced by the `hack/bulid-cross.sh` script.